glossary/project.md

```diff
@@ -1 +1,3 @@
-A distinct thing we are developing as part of a [product](product.md). [Achievables](achievable.md) are often part of a project.		
+A distinct thing we are developing as part of a [product](product.md). [Achievables](achievable.md) are often part of a project.
+
+**As of August 19, 2016, projects have been replaced by [milestones](milestone.md).** 
```

Comment:
I think as long as there's still a little bit of 'projects' left in our system (in the database, for example) it can be handy to have it there...